src/qml/imports/QFieldControls/+Qt5/QFieldCamera.qml

```diff
@@ -376,5 +384,26 @@ Popup {
         }
       }
     }
+
+    QfToolButton {
+      id: geotagButton
+
+      anchors.left: parent.left
+      anchors.leftMargin: 4
+      anchors.top: backButton.bottom
+      anchors.topMargin: 4
+
+      iconSource: positionSource.active ? Theme.getThemeIcon("ic_geotag_24dp") : Theme.getThemeIcon("ic_geotag_missing_24dp")
+      iconColor: positionSource.active && settings.geoTagging ? Theme.mainColor : "white"
```

Comment:
```suggestion
      iconColor: positionSource.active && settings.geoTagging ? Theme.mainColor : Theme.lightColor
```

Isn't it better to avoid mixing themed and hardcoded colors?

Reply:
In this case, not really. I mean, we could have Theme.white, but I feel like it's not gaining much.

This specific color doesn't change across our light/dark theme either.

If we want to say have something like Theme.defaultToolButttonColor (being white) I suggest we do that in a separate PR as it'll touch a dozen QML files or so.